index.html

```diff
@@ -1277,20 +1277,20 @@ <h2>Relationship to Verifiable Credentials</h2>
 
         <p>
 Finally, implementers are also urged to understand that there is a difference
-between the <a data-cite="CONTROLLER-DOCUMENT#dfn-revoked">revocation time</a>
-and <a data-cite="CONTROLLER-DOCUMENT#defn-vm-expires">expiration time</a> for a
-[=verification method=], and the revocation information associated with a
-[=verifiable credential=]. The
-<a data-cite="CONTROLLER-DOCUMENT#dfn-revoked">revocation time</a> and
-<a data-cite="CONTROLLER-DOCUMENT#defn-vm-expires">expiration time</a> for a
+between the revocation information associated with a [=verifiable credential=],
+and the <a data-cite="CONTROLLER-DOCUMENT#dfn-revoked">revocation</a>
+and <a data-cite="CONTROLLER-DOCUMENT#defn-vm-expires">expiration</a> times
+for a [=verification method=]. The
+<a data-cite="CONTROLLER-DOCUMENT#dfn-revoked">revocation</a> and
+<a data-cite="CONTROLLER-DOCUMENT#defn-vm-expires">expiration</a> times for a
 [=verification method=] are expressed using the `revocation` and `expires`
-properties, respectively, and are related to events such as a private key being
-compromised or expiring and can provide timing information which might reveal
+properties, respectively; are related to events such as a private key being
+compromised or expiring; and can provide timing information which might reveal
 details about a controller such as their security practices or when they might
```

Comment:
Missed this the first time...
```suggestion
details about a controller, such as their security practices or when they might
```